src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/ArtifactsSizeTests/centos.9-x64.txt

```diff
@@ -278,7 +280,7 @@ packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/it/Microsoft.Extens
 packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/ja/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 8192
 packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/ko/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 8192
 packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/Microsoft.AspNetCore.App.Analyzers.dll: 240128
-packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/Microsoft.AspNetCore.App.CodeFixes.dll: 29184
+packs/Microsoft.AspNetCore.App.Ref/x.y.z/analyzers/dotnet/cs/Microsoft.AspNetCore.App.CodeFixes.dll: 45056
```

Comment:
For the aspnetcore differences, the diff between this build and the last showed that SDK had updated dependencies from aspnetcore, one of which was Microsoft.AspNetCore.App.Ref. Based on this, I think that these differences are legitimate product changes.